level-2/N개의-최소공배수.py

```diff
@@ -0,0 +1,18 @@
+def gcd(a, b):
+    if a > b:
+        a, b = b, a
+    while b%a != 0:
+        b, a = a, b%a
+    return a
+
+def solution(arr):
+    answer = 1
+    arr = sorted(arr)
```

Comment:
Think about the space complexity of this algoithm. If you haven't heard about it, I think it is a good time to understand this concept. 

Do you know the difference between `sort` and `sorted`? Which one is better in terms of space complexity and why? When should I use one with the more space complexity?

Reply:
Nevermind. This requires too much information to answer... You don't have to answer it. Just keep in mind that `sorted` returns a new array.